Description: Fix binary-or-shlib-defines-rpath Lintian warning.
Author: Benjamin Drung <bdrung@ubuntu.com>
Forwarded: No

Index: debian.git/src/plugins/sid/wscript
===================================================================
--- debian.git.orig/src/plugins/sid/wscript	2010-02-19 18:43:54.000000000 +0100
+++ debian.git/src/plugins/sid/wscript	2010-02-20 12:50:08.312900440 +0100
@@ -14,13 +14,6 @@
     # extract what's inside builders pkg-config variable
     builders = conf.check_cfg(package="libsidplay2", args="--variable=builders").strip()
 
-    # It looks funny that this is LIBPATH_sidplay but it is correct
-    # since we explcitly set the path to builders for resid it will
-    # find resid in the list even though we still need to -rpath it.
-    if not realpath(builders) in [realpath(i) for i in conf.env['LIBPATH_sidplay']]:
-        conf.env['LINKFLAGS_resid'] = ["-Wl,-rpath=%s" % builders]
-        conf.env["LIBPATH_resid"] = [builders]
-
     if not conf.check_cxx(lib="resid-builder", header_name="sidplay/builders/resid.h", uselib_store="resid", uselib="resid"):
         return False
 
