From: Benjamin Drung <bdrung@ubuntu.com>
Subject: [PATCH] fixes/detabify

De-tabify COPYING.

Signed-off-by: Benjamin Drung <bdrung@ubuntu.com>

---
 COPYING |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/COPYING b/COPYING
index ae62935..6994b1c 100644
--- a/COPYING
+++ b/COPYING
@@ -24,7 +24,7 @@ manpages (src/clients/cli/xmms2.1, src/clients/et/xmms2-et.1,
           src/clients/launcher/xmms2-launcher.1,
           src/clients/mdns/avahi/xmms2-mdns-avahi.1,
           src/xmms/xmms2d.1
-	   src/clients/nycli/nyxmms2.1):
+          src/clients/nycli/nyxmms2.1):
         Licensed under BSD license
 
 src/plugins/mad/*.[ch]:
@@ -94,10 +94,10 @@ src/plugins/normalize/compress*:
         Copyright (C) 2002-2003 trikuare studios (http://trikuare.cx)
 
 src/plugins/airplay/rtsp*:
-	Copyright (C) <2005> Wim Taymans <wim@fluendo.com>
+        Copyright (C) <2005> Wim Taymans <wim@fluendo.com>
 
 src/plugins/airplay/airplay.c:
-	Copyright (C) 2005-2006 Mohsin Patel <mohsin.patel@gmail.com>
+        Copyright (C) 2005-2006 Mohsin Patel <mohsin.patel@gmail.com>
 
 src/includepriv/xmmspriv/xmms_list.h:
         Copyright (C) 1995-1997  Peter Mattis, Spencer Kimball and Josh MacDonald
-- 
tg: (947d326..) fixes/detabify (depends on: master)
